CHANGELOG.md

```diff
@@ -29,6 +29,11 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#2123](https://github.com/realm/SwiftLint/issues/2123)
 
+* Add `toggle_bool` opt-in rule which suggests using `Bool.toggle()` over 
+  `someBool = !someBool`. Requires Swift 4.2.
```

Comment:
Please make sure there are two trailing spaces on the last line describing the change. See [CONTRIBUTING.md](https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes)

Reply:
Whoops, must have missed that one. 😅